src/main/resources/templates/index.html

```diff
@@ -188,12 +189,13 @@
             });
         }
 
-        function init(){
+        $(document).ready(function(){
             initDialog();
             closeDialog();
             refreshSel();
             refresh();
-        };
+        });
```

Comment:
Not a best practice to call a function without the document being ready and without context. The jquery function "$(document).ready()" serves this purpose, thus reducing technical debt.